packages/react-core/src/demos/DataListDemo.md

```diff
@@ -418,3 +424,899 @@ class ExpandableDataList extends React.Component {
+        Separated Action
+      </OverflowMenuDropdownItem>
+    ];
+
+    const toolbarItems = (
+      <React.Fragment>
+        <ToolbarItem variant="bulk-select">{this.buildSelectDropdown()}</ToolbarItem>
+        <ToolbarItem breakpoint="xl">{this.buildFilterDropdown()}</ToolbarItem>
+        <ToolbarItem variant="overflow-menu">
+          <OverflowMenu breakpoint="md">
+            <OverflowMenuItem>
+              <Button variant="primary">Create a project</Button>
+            </OverflowMenuItem>
+            <OverflowMenuControl hasAdditionalOptions>
+              <Dropdown
+                onSelect={this.onToolbarKebabDropdownSelect}
+                toggle={(toggleRef) => (
+                  <MenuToggle
+                    ref={toggleRef}
+                    aria-label="Toolbar kebab overflow menu"
+                    variant="plain"
+                    onClick={this.onToolbarKebabDropdownToggle}
+                    isExpanded={isLowerToolbarKebabDropdownOpen}
+                  >
+                    <EllipsisVIcon />
+                  </MenuToggle>
+                )}
+                isOpen={isLowerToolbarKebabDropdownOpen}
+                onOpenChange={(isOpen) => this.setState({ isLowerToolbarKebabDropdownOpen: isOpen })}
+              >
+                <DropdownList>{toolbarKebabDropdownItems}</DropdownList>
+              </Dropdown>
+            </OverflowMenuControl>
+          </OverflowMenu>
+        </ToolbarItem>
+        <ToolbarItem variant="pagination" align={{ default: 'alignRight' }}>
+          {this.renderPagination()}
+        </ToolbarItem>
+      </React.Fragment>
+    );
+
+    const filtered =
+      filters.products.length > 0
+        ? res.filter((card) => {
+            return filters.products.length === 0 || filters.products.includes(card.name);
+          })
+        : res;
+
+    const icons = {
+      pfIcon,
+      activeMQIcon,
+      sparkIcon,
+      avroIcon,
+      azureIcon,
+      saxonIcon,
+      dropBoxIcon,
+      infinispanIcon,
+      restIcon,
+      swaggerIcon
+    };
+
+
+    return (
+      <ToolbarFilter categoryName="Products" chips={filters.products} deleteChip={this.onDelete}>
+        <Select
+          aria-label="Products"
+          role="menu"
+          toggle={(toggleRef) => (
+            <MenuToggle ref={toggleRef} onClick={this.onToolbarDropdownToggle} isExpanded={isLowerToolbarDropdownOpen}>
+              My Project
+              {filters.products.length > 0 && <Badge isRead>{filters.products.length}</Badge>}
+            </MenuToggle>
+          )}
+          onSelect={this.onNameSelect}
+          onOpenChange={(isOpen) => {
+            this.setState(() => ({
+              isLowerToolbarDropdownOpen: isOpen
+            }));
+          }}
+          selected={filters.products}
+          isOpen={isLowerToolbarDropdownOpen}
+        >
+          {filterDropdownItems}
+        </Select>
+      </ToolbarFilter>
+    );
+  }
+
+  render() {
```

Comment:
You have two render functions defined in this demo. This one, and the one on line 885. I think you only need one. If you opt for the first one, you can clean up many of the variables pulled out of state on line 902. whichever render method is the one you intend to use, you'll need to clean up many unused imports.

A fast way to check for unused imports is to open the demo (once you have it running locally) in a codesandbox by hitting this button below the demo
![Screenshot 2023-05-10 at 8 52 40 AM](https://github.com/patternfly/patternfly-react/assets/26305082/4870ddee-b9d9-4904-91f4-3d7768706566)

Reply:
Just wanted to say, I will be finishing up this Thursday as it is my last day.